### PR TITLE
Players and card display table

### LINKED ISSUE #67: [Feature 22] Displaying info table
Add a table that displays the other players and their respective number of cards. Updates with the game, as players play cards, their number of cards updates.

### PR DESCRIPTION
**Pull request by meiiph**
Resolves #67 by adding a table to the gameplay window that updates with each turn.

### CODE REVIEW

**Review by lilyxmeng — APPROVED** [review]

File: src/main/java/ui/windows/game/GameDisplay.java [hunk 1]
```diff
@@ -42,6 +49,7 @@ public GameDisplay(GameDelegator gameDelegator, CardDelegator cardDelegator) {
         this.drawDelegator = gameDelegator.getDrawDelegator();
         this.cardDelegator = cardDelegator;
         this.currentCard = new DrawnCard(Suit.HEART, Rank.ACE);
+        this.playerInfo = new Object[][]{{"", 0}, {"", 0}, {"", 0}, {"", 0}, {"", 0}, {"", 0}};
```
Comment (SMateen21) [inline src/main/java/ui/windows/game/GameDisplay.java:52]: Maybe you could make the 0s all Strings and have this as String[][], you'll have to do conversions back and forth between String and Integer though. It's your call though.

**Review by SMateen21 — APPROVED** [review]
Looks good. Thanks for adding the missing documentation

**Review by nyan-ko — APPROVED** [review]

File: src/main/java/ui/windows/game/GameDisplay.java (hunk 1 above)
Comment (meiiph) [reply src/main/java/ui/windows/game/GameDisplay.java:52]: That's a good idea, that way the 0s won't show up when there's no players to take up that row in the game. I will try to implement this now. :)

**Review by nyan-ko — DISMISSED** [review]
changes look good

**Review by lilyxmeng — APPROVED** [review]

**Review by SMateen21 — DISMISSED** [review]
Looks good to me

**Review by nyan-ko — APPROVED** [review]
LGTM

**Review by lilyxmeng — APPROVED** [review]

**Review by SMateen21 — APPROVED** [review]

**Pull request merged**